### DISCUSSION TITLE
equivalent solution for papers/docs

### DISCUSSION THREAD

**Discussion by vasnt** [Shelf-nu/shelf.nu#211] [Q&A]
I ocassionally struggle to find documents. is there any tool available that can help to manage/relocate hard-copy documents/papers?

**Comment by jurrejansen** [comment]
Hi @vasnt

I may need some more context on the way you are currently archiving your folders / documents but you should definitely be able to use Shelf to manage this.

You can tag your folders / documents by sticking a QR asset label sticker on them, scan the code and create an **asset** entry in your Shelf environment.

Use **categories** and **tags** to organise your documents and folders to make your asset index overview-able and easy to filter through.

I can imagine you're storing you folders / documents in file cabinets or something similar. Within your Shelf environment you can create **locations** and link your assets to a certain location. This should help you with finding out in which cabinet, and maybe even drawer (by using tags), the folder / document is stored.

If you tag your folders / documents with the QR sticker, you will be able to easily change the location in the environment after you relocated it physically.

Currently a single user Shelf environment is completely free. I would love for you to try it out, if you haven't already, and see if Shelf fits your needs. https://app.shelf.nu/join 

Let me know if you any specific questions. Have a great day!
